js/Heap.js

```diff
@@ -1,7 +1,8 @@
 // Libraries
-import {NativeModules, Platform} from 'react-native';
-import Package from 'react-native-package';
+import { NativeModules, Platform } from "react-native";
+import Package from "react-native-package";
```

Comment:
Let's keep single quotes where we don't need double quotes (same with changes below).